### ISSUE TITLE
Add a margin between the end of the content of each page and the disqus module

### ISSUE THREAD

**Issue by lionel-m** [contao/docs-archive#261]
If possible

**Comment by aschempp** [comment]
Currently I can't (or I don't know how to) add custom CSS to the book. Obviously this is somehow possible, but I didn't find the time for the details yet ;-)

**Comment by lionel-m** [comment]
We can add custom style sheets with the `book.json` file

``` json
{
    "styles": {
        "website": "styles/website.css"
    }
}
```

http://help.gitbook.com/format/configuration.html#styles

**Comment by aschempp** [comment]
Fixed in 435d203dd984a762ef02631723ce287e1300af2d
